### PR TITLE
feat(connector): add kook connector

### PR DESCRIPTION
**Pull request by Misaka-L**
## Summary

feat(connector): add kook connector

## Testing

- scope: get_user_info

image
image
image

## Checklist

- [x] `.changeset`
- [x] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Comment by Misaka-L** [comment]
Sorry for force-push multiple times, I didn't familiar with signature commit

**Comment by simeng-li** [comment]
no worries.  Thanks for the contribution. @wangsijie @darcyYe can you guys take a look at it?

File: packages/connectors/connector-kook/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- [KOOK 连接器](#kook-连接器)
+  - [配置你的 KOOK 连接器](#配置你的-kook-连接器)
+    - [配置类型](#配置类型)
+  - [测试 KOOK 连接器](#测试-kook-连接器)
+  - [参考](#参考)
+
+
+## Get started
+
+If you don't know the concept of the connector or don't know how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```
Comment (wangsijie) [inline packages/connectors/connector-kook/README.md:24]: To align with other connectors, we can remove this line.

File: packages/connectors/connector-kook/logo.svg
Comment (wangsijie) [inline packages/connectors/connector-kook/logo.svg:1]: Is there a logo for dark mode?

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/connectors/connector-kook/logo.svg
Comment (Misaka-L) [reply packages/connectors/connector-kook/logo.svg:1]: No, KOOK use same icon in both dark mode and light mode.

File: packages/connectors/connector-kook/README.md (hunk 1 above)
Comment (Misaka-L) [reply packages/connectors/connector-kook/README.md]: removed

Comment (Misaka-L) [reply packages/connectors/connector-kook/README.md]: removed

**Review by darcyYe — APPROVED** [review]
Over all LGTM, would you like to update the building tool since we've just migrate to `tsup`. You can refer to [this PR](https://github.com/logto-io/logto/pull/6355), the change is straightforward.

**Comment by Misaka-L** [comment]
> Over all LGTM, would you like to update the building tool since we've just migrate to `tsup`. You can refer to [this PR](https://github.com/logto-io/logto/pull/6355), the change is straightforward.

I will migrate it later. Thanks

**Comment by Misaka-L** [comment]
> Over all LGTM, would you like to update the building tool since we've just migrate to `tsup`. You can refer to [this PR](https://github.com/logto-io/logto/pull/6355), the change is straightforward.

I have migrate to tsup and I would like to know that is this pull request still on tracking?

**Review by gao-sun — APPROVED** [review]

**Pull request merged**